### PR TITLE
Change 'typedef' to 'using', add curly braces

### PR DESCRIPTION
**Pull request by the-raspberry-pi-guy**
Simple PR to change:
- `typedef` to `using` when defining the Vector6d, as there are advantages to the `using` syntax: https://stackoverflow.com/a/32959200
- Add some curly braces to the success/failure scenarios, as adding further statements after these tripped me up in the same way as: https://stackoverflow.com/a/2125078

### CODE REVIEW

**Comment by hrp2-14** [comment]
Hi ! This project doesn't usually accept pull requests on master. If this wasn't intentionnal, you can change the base branch of this pull request to devel (No need to close it for that). Best, a bot.

**Comment by proyan** [comment]
Hi @the-raspberry-pi-guy
Thanks for contributing to pinocchio :)
Unfortunately, type aliases are a C++11 feature, and we still maintain compatibility with C++98 systems.

**Comment by the-raspberry-pi-guy** [comment]
Hi @proyan 

My apologies - I missed checking that :) No worries about type aliases. This becomes a very minor PR then, but I would still recommend including the curly braces as part of the beginner tutorials :)

Thanks

**Comment by jcarpent** [comment]
> This becomes a very minor PR then, but I would still recommend including the curly braces as part of the beginner tutorials :)

Thanks for your contribution. Why do you say that?

File: examples/inverse-kinematics.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@ int main(int /* argc */, char ** /* argv */)
   J.setZero();
 
   bool success = false;
-  typedef Eigen::Matrix<double, 6, 1> Vector6d;
+  using Vector6d = Eigen::Matrix<double, 6, 1>;
```
Comment (jcarpent) [inline examples/inverse-kinematics.cpp:26]:
```suggestion
  typedef Eigen::Matrix<double, 6, 1> Vector6d;
```

**Comment by the-raspberry-pi-guy** [comment]
> > This becomes a very minor PR then, but I would still recommend including the curly braces as part of the beginner tutorials :)
> 
> Thanks for your contribution. Why do you say that?

It is a matter of preference, but as a matter of good coding style I think it is better to add curly braces here. I used this example and added a statement in what I thought was one of the nested structures, only to be caught out by the lack of curly braces. Curly brace use/not use should be consistent, hence just pointing it out. https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces/2125078#2125078

At your discretion though of course.

**Comment by jcarpent** [comment]
Thanks.

**Pull request merged**